backend/src/main/java/com/benine/backend/database/MySQLDatabase.java

```diff
@@ -266,6 +268,38 @@ public void addCamera(int id, String ip) {
   }
 
   @Override
+  public ArrayList<Camera> getAllCameras() throws SQLException {
```

Comment:
Maybe this method is a bit long. Is it possible to split it into smaller methods?